cluster-controller/src/main/java/io/strimzi/controller/cluster/ClusterControllerConfig.java

```diff
@@ -20,24 +20,29 @@
     public static final String STRIMZI_NAMESPACE = "STRIMZI_NAMESPACE";
     public static final String STRIMZI_CONFIGMAP_LABELS = "STRIMZI_CONFIGMAP_LABELS";
     public static final String STRIMZI_FULL_RECONCILIATION_INTERVAL = "STRIMZI_FULL_RECONCILIATION_INTERVAL";
+    public static final String STRIMZI_OPERATION_TIMEOUT = "STRIMZI_OPERATION_TIMEOUT";
 
     public static final long DEFAULT_FULL_RECONCILIATION_INTERVAL = 120_000; // in ms (2 minutes)
+    public static final long DEFAULT_OPERATION_TIMEOUT = 60_000; // in ms (1 minute)
```

Comment:
I like the Kafka habit of including the unit in the same: `DEFAULT_OPERATION_TIMEOUT_MS` would makes the unit unambiguous at the use site, and there's then no need for the comment. This can be deferred to another PR if you prefer.